hub-chart/values.yaml

```diff
@@ -0,0 +1,59 @@
+# Default values for KubeStellar-Hub.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+# Container/chart image versions. These values are set during the chart release process,
+# please do not change them unless you know what your are doing.
+HELM_VERSION: "3.14.0"
+KUBECTL_VERSION: "1.29.3"
+KUBESTELLAR_VERSION: "0.23.0-alpha.2"
+CLUSTERADM_VERSION: "0.8.2"
+OCM_STATUS_ADDON_VERSION: "0.2.0-rc8"
+OCM_TRANSPORT_PLUGIN_VERSION: "0.1.7"
+
+
+# Control controller log verbosity
+# The global VERBOSITY value will be sued for all controller unless a specific controller verbosity override is specified
+VERBOSITY: 2 # default global value for all controllers
+# Specific controller verbosity overrides:
+# KUBEFLEX_VERBOSITY_OVERRIDE: 6
+# KUBESTELLAR_VERBOSITY_OVERRIDE: 6
+# CLUSTERADM_VERBOSITY_OVERRIDE: 6
+# OCM_STATUS_ADDON_VERBOSITY_OVERRIDE: 6
+# OCM_TRANSPORT_PLUGIN_VERBOSITY_OVERRIDE: 6
+
+
+# KubeFlex override values
+kubeflex-operator:
+  install: true
+  # installPostgreSQL: true # automatically install the appropriate version of PostgreSQL required by KubeFlex
+  # isOpenShift: false # set this variable to true when installing the chart in an OpenShift cluster
+  # domain: localtest.me # used to set the Control Planes DNS in a Kind cluster installation
+  # externalPort: 9443 # used to set the port to access the Control Planes API
+  # hostContainer: kubeflex-control-plane # set the name of kubeflex control plane
```

Comment:
If I understand correctly, this gets set to the name of a container rather than the KubeFlex name of a control plane. I do not understand the general story (see https://github.com/kubestellar/kubeflex/issues/250), but the wording here should be clarified. I suggest something like "the name of the container that runs the KubeFlex hosting cluster". Ideally this would include a pointer to the KubeFlex documentation for this, but there is none (as far as I can tell, @pdettori).

A little more precisely, this is an opportunity to override the default for a "value" of the kubeflex helm chart. I would explicitly say this.

Reply:
ok, done